recipe/meta.yaml

```diff
@@ -7,9 +7,7 @@ package:
 
 source:
   url: https://github.com/numpy/numpy/releases/download/v{{ version }}/numpy-{{ version }}.tar.gz
-  sha256: 766e09248298e3ad4ae4a805159f358610bbe7dcc7b4a14e5df2128c05655b80
-  patches:
-    - numpy_distutils_ldflags.diff
```

Comment:
@isuruf can we just remove this or should I refresh it?

Reply:
It can be removed

Reply:
Thanks!